### DISCUSSION TITLE
Feature Request - OAuth2 or LDAP support

### DISCUSSION THREAD

**Discussion by primeral** [gaseous-project/gaseous-server#435] [Ideas]
Great project! My feature request is to have support for either OAuth2 or LDAP. Ideally I want gaseous to integrate with Authentik for centralized user management. Thanks and keep up the good work.

**Comment by michael-j-green** [comment]
Thankyou for your feedback and comments! :)

This is actually planned for the version I'm currently working on now (see https://github.com/gaseous-project/gaseous-server/issues/342).

I plan to start getting into that particular feature soon-ish.

**Comment by michael-j-green** [comment]
Merged in the form of OIDC as part of PR: https://github.com/gaseous-project/gaseous-server/issues/489
